featuretools/feature_base/feature_base.py

```diff
@@ -626,3 +697,96 @@ def _check_feature(feature):
         return feature
 
     raise Exception("Not a feature")
+
+
+def _find_path(start_entity_id, end_entity_id, es, backward=False):
```

Comment:
Since here we are enforcing that the path is unique the algorithm is different and more complicated than that in `EntitySet`. There should still be some logic that can be shared, but I thought it would be easier to look at this in isolation first. Then, once we are confident in the implementation here I can merge it into `EntitySet`.